feat: official resources section (#967)
italia/eidas-it-wallet-docs: docs/en/functionalities.rst

    IT-Wallet ID official graphic asset on a light background 

> The IT-Wallet ID Official Resources are provided in the related :ref:`official-resources:Official Resources` Section. Additional documentation on the IT-Wallet ID graphic assets is available in the Brand Manual, as indicated in the :ref:`official-resources:Official Resources` section.  

Reviewer: ```suggestion
The IT-Wallet ID Official Resource is provided in the related :ref:`official-resources:Official Resources` Section. Additional documentation on the IT-Wallet ID graphic asset is available in the Brand Manual, indicated in the :ref:`official-resources:Official Resources` section.  
```

- The Authentication Button MUST state "Login with IT-Wallet";

> - The Authentication Button SHOULD always be accompanied by an external link (e.g., "Learn more") that links to the official website of the IT-Wallet System, as indicated in the :ref:`official-resources:Official Resources` section; 

Reviewer: ```suggestion
- The Authentication Button SHOULD always be accompanied by an external link (e.g., "Learn more") that links to the official website of the IT-Wallet System, indicated in the :ref:`official-resources:Official Resources` section; 
```

**"Login with IT-Wallet" button - html code**  

The button is available in three variants of dimensions (S - default / M / L ) as per the .Italia Design System, and in "get" (call to an external page) and "post" (form inside the button) formats. In addition to the variants of dimensions, are provided two variants of button with fixed length, to be used in situations where it's preferable to keep consistency between similar buttons:

- the fixed-width version with icon on the left and centered text;
- the fixed‑width version with icon and text centered.

> The Official Resource of the Authentication Button is available in the :ref:official-resources:Official Resources of these Technical Specification.

Reviewer: ```suggestion
The Official Resource of the Authentication Button is available in the :ref:`official-resources:Official Resources` of these Technical Specification.
```

**"Login with IT-Wallet" button - html code**  

> The button is available in three variants of dimensions (S - default / M / L ) as per the .Italia Design System, and in "get" (call to an external page) and "post" (form inside the button) formats. In addition to the variants of dimensions, are provided two variants of button with fixed length, to be used in situations where it's preferable to keep consistency between similar buttons:

Reviewer: ```suggestion
The button is available in three variants of dimensions (S - default / M / L ) as per the .Italia Design System, and in "get" (call to an external page) and "post" (form inside the button) formats. In addition to the variants of dimensions, are provided two variants of button with fixed width, to be used in situations where it's preferable to keep consistency between similar buttons:
```

 

**"Login with IT-Wallet" button – svg**

> For further information on the Authentication button refer to the Brand Manual, as indicated in the :ref:`official-resources:Official Resources` section. Authentication button Official Resource is available in the :ref:`official-resources:Official Resources` section of these Technical Specifications. 

Reviewer: ```suggestion
For further information on the Authentication button refer to the Brand Manual, indicated in the :ref:`official-resources:Official Resources` section. The Authentication button Official Resource is available in the :ref:`official-resources:Official Resources` section of these Technical Specifications. 
```
